InAppUtils/InAppUtils.m

```diff
@@ -35,6 +35,10 @@ - (void)paymentQueue:(SKPaymentQueue *)queue
                 NSString *key = RCTKeyForInstance(transaction.payment.productIdentifier);
                 RCTResponseSenderBlock callback = _callbacks[key];
                 if (callback) {
+                    if (transaction.error.code == SKErrorPaymentCancelled) {
```

Comment:
`RCTJSErrorFromNSError` is still used below, and would presumably still fail for every other error other than cancel?